diff --git a/node_modules/speedlify-score/speedlify-score.js b/node_modules/speedlify-score/speedlify-score.js
index 9c7d0d7..1fcb4c8 100644
--- a/node_modules/speedlify-score/speedlify-score.js
+++ b/node_modules/speedlify-score/speedlify-score.js
@@ -76,20 +76,20 @@ class SpeedlifyScore extends HTMLElement {
 	align-items: center;
 	justify-content: center;
 	border-radius: 50%;
-	border: 0.15384615em solid currentColor; /* 2px /13 */
+	border: 0.15384615em solid #666; /* 2px /13 */
 	color: var(--_circle, #666);
 }
 .circle-good {
-	color: #088645;
-	border-color: #0cce6b;
+	color: #088846;
+	border-color: #088846;
 }
 .circle-ok {
-	color: #ffa400;
-	border-color: currentColor;
+	color: #A56900;
+	border-color: #A56900;
 }
 .circle-bad {
-	color: #ff4e42;
-	border-color: currentColor;
+	color: #EC0D00;
+	border-color: #EC0D00;
 }
 .meta {
 	display: flex;
@@ -106,13 +106,13 @@ class SpeedlifyScore extends HTMLElement {
 	line-height: 1;
 }
 .rank-change.up {
-	color: green;
+	color: #088846;
 }
 .rank-change.up:before {
 	content: "⬆";
 }
 .rank-change.down {
-	color: red;
+	color: #EC0D00;
 }
 .rank-change.down:before {
 	content: "⬇";
@@ -160,7 +160,7 @@ class SpeedlifyScore extends HTMLElement {
 	async init() {
 		if(this.rawData) {
 			let data = JSON.parse(this.rawData);
-			this.setDateAttributes(data);
+			// this.setDateAttributes(data);
 			this._initTemplate(data);
 			return;
 		}
@@ -187,18 +187,18 @@ class SpeedlifyScore extends HTMLElement {
 		}
 
 		let data = await urlStore.fetchData(this.speedlifyUrl, hash);
-		this.setDateAttributes(data);
+		// this.setDateAttributes(data);
 
 		this._initTemplate(data, forceRerender);
 	}
 
-	setDateAttributes(data) {
-		if(!("Intl" in window) || !Intl.DateTimeFormat || !data.timestamp) {
-			return;
-		}
-		const date = new Intl.DateTimeFormat().format(new Date(data.timestamp));
-		this.setAttribute("title", `Results from ${date}`);
-	}
+	// setDateAttributes(data) {
+	// 	if(!("Intl" in window) || !Intl.DateTimeFormat || !data.timestamp) {
+	// 		return;
+	// 	}
+	// 	const date = new Intl.DateTimeFormat().format(new Date(data.timestamp));
+	// 	this.setAttribute("title", `Results from ${date}`);
+	// }
 
 	getScoreClass(score) {
 		if(score === "" || score === undefined) {
